### PR TITLE
nwg-launchers: update to 0.7.1.1

### PR DESCRIPTION
**Pull request by cinerea0**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/nwg-launchers/template [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 hostmakedepends="pkg-config"
 makedepends="gtkmm-devel gtk-layer-shell-devel json-c++"
 short_desc="GTK-based launchers for window managers"
 maintainer="Érico Nogueira <ericonr@disroot.org>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/nwg-piotr/nwg-launchers"
-distfiles="${homepage}/archive/v${version}.tar.gz"
-checksum=8be219ef7e8aead08b46215c9c545b1bf9d108cefee580ffcf297553dee9dfaf
+distfiles="https://github.com/nwg-piotr/nwg-launchers/archive/refs/tags/v${version}.tar.gz"
+checksum=3700fe67870ecead091abe672c93ce01a6351cb1e0be1904233050c22e86a7c4
```
Comment (sgn) [inline srcpkgs/nwg-launchers/template:13]:
```suggestion
checksum=3700fe67870ecead091abe672c93ce01a6351cb1e0be1904233050c22e86a7c4

post_patch() {
    rm -f make_readme.py
    ln -s /bin/true make_readme.py
}
```

File: srcpkgs/nwg-launchers/template [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
+
+post_patch() {
+    rm -f make_readme.py
+    ln -s /bin/true make_readme.py
```
Comment (sgn) [inline srcpkgs/nwg-launchers/template:17]:
```suggestion
	echo "" >make_readme.py
```

Comment (classabbyamp) [reply srcpkgs/nwg-launchers/template]: or `: > make_readme.py`

File: srcpkgs/nwg-launchers/template [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
 maintainer="Érico Nogueira <ericonr@disroot.org>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/nwg-piotr/nwg-launchers"
-distfiles="${homepage}/archive/v${version}.tar.gz"
-checksum=8be219ef7e8aead08b46215c9c545b1bf9d108cefee580ffcf297553dee9dfaf
+distfiles="https://github.com/nwg-piotr/nwg-launchers/archive/refs/tags/v${version}.tar.gz"
+checksum=3700fe67870ecead091abe672c93ce01a6351cb1e0be1904233050c22e86a7c4
+
+post_patch() {
+	echo "" >make_readme.py
```
Comment (sgn) [inline srcpkgs/nwg-launchers/template:16]:
```suggestion
post_configure() {
	: >make_readme.py
	cp README.md build
```

**Pull request merged**